README.rst

```diff
@@ -12,3 +12,22 @@ research projects.
 The hosted documentation can be found at https://vivarium-research.readthedocs.io/en/latest/.
 
 Information on how to contribute to vivarium research docs can be found at https://vivarium-research.readthedocs.io/en/latest/onboarding_resources/contributing/index.html. 
+
+Use the following steps to build the docs locally::
+
+
+.. _installation:
+::
+
+   $> conda create -y --name=vivarium_research python=3.13 graphviz
```

Comment:
```suggestion
   $> conda create -y --name=vivarium_research python=3.13 graphviz pandoc
```
Might as well add `pandoc` now so we're ready for when https://github.com/ihmeuw/vivarium_research/pull/1579 is merged.